src/inspector_agent.cc

```diff
@@ -380,6 +388,53 @@ void AgentImpl::WaitForDisconnect() {
   inspector_->runMessageLoopOnPause(0);
 }
 
+String16 toProtocolString(v8::Local<v8::Value> value) {
+  if (value.IsEmpty() || value->IsNull() || value->IsUndefined() ||
+      value->IsString()) {
+    return String16();
+  }
+  v8::Local<v8::String> string_value = v8::Local<v8::String>::Cast(value);
+  std::unique_ptr<UChar[]> buffer(new UChar[string_value->Length()]);
```

Comment:
I suspect this may be using the `std::unique_ptr` leaking out of the v8_inspector headers. A fake stl implementation was added to allow v8_inspector to work w/ Node.js `v6.x`. I can build this on my mac, with the Node.js build default of `-mmacosx-version-min=10.7`.